### PR TITLE
Add internal merges for core repos and aspnet/extensions

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@dagood PTAL

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -18,18 +18,24 @@
...
-    // A 'build' that mirrors changes from github into the dnceng intance of VSTS
-    "github-dnceng-vsts-mirror": {
+    // A 'build' that mirrors changes from github into the dnceng intance of Azure DevOps
+    "github-dnceng-azdo-mirror": {
       "vsoInstance": "dnceng.visualstudio.com",
       "vsoProject": "internal",
       "buildDefinitionId": 16
     },
+    // A 'build' that mirrors changes from github into the dnceng intance of Azure DevOps
+    "github-dnceng-azdo-internal-merge-mirror": {
```
Comment (dagood) [inline Maestro/subscriptions.json]: From the PR title I can guess what it does, but the def names and comments seem mismatched making the diff confusing. Is this similar to `dotnet-devdiv-branch-merge-mirror`, should one of the names be clarified to represent that?

Comment (mmitche) [reply Maestro/subscriptions.json]: Updated

Comment (dagood) [inline Maestro/subscriptions.json]: Thanks, comment still looks wrong though.

**Pull request merged**